### DISCUSSION TITLE
Is there a way to use this library with `import` instead of `require()`?

### DISCUSSION THREAD

**Discussion by asimjb1200** [XRPLF/xrpl.js#1416] [Q&A] [answered]
I have a small project I am working on and when I try:
```javascript
import ripple from 'ripple-lib';
const api = new ripple.RippleAPI({
    server: 'wss://s1.ripple.com' // Public rippled server
});
```

I get an error saying: `TypeError: Cannot read property 'RippleAPI' of undefined` after I compile my ts code and try to run it.

**Comment by intelliot** [comment, marked as answer]
@asimjb1200 the `ripple` object is only used for the browser build of this library. When using Node.js, instead try:
```javascript
import {RippleAPI} from 'ripple-lib';
const api = new RippleAPI({
    server: 'wss://s1.ripple.com' // Public rippled server
});
```
